tests/test_requests.py

```diff
@@ -1452,3 +1451,8 @@ def test_vendor_aliases():
 
     with pytest.raises(ImportError):
         from requests.packages import webbrowser
+
+
+
+if __name__ == '__main__':
+    unittest.main()
```

Comment:
Ok that was not on purpose, I must have pasted it without noticing it was there. Fixed